compute/instances/troubleshooting/fix-dns-routed-ipv6-only-debian-bullseye.mdx

```diff
@@ -86,7 +86,7 @@ In this situation, proceed with the next section to install `netplan` before app
     All steps below require super-user (`root`) privileges.
 </Message>
 
-1. *(optional)** If, **and only if**,  your Instance is already booted using a routed IPv6-only setup, you need to temporarily configure your DNS resolver so that it can reach the Debian repositories, in order to install `netplan`. The following uses Google's DNS server:
+1. *(optional)* If, **and only if**,  your Instance is already booted using a routed IPv6-only setup, you need to temporarily configure your DNS resolver so that it can reach the Debian repositories, in order to install `netplan`. The following uses Google's DNS server:
```

Comment:
```suggestion
1. *(optional)* If, **and only if**, your Instance is already booted using a routed IPv6-only setup, you need to temporarily configure your DNS resolver so that it can reach the Debian repositories, in order to install `netplan`. The following uses Google's DNS server:
```